packages/fdc3-agent-proxy/test/support/responses/RegisterListeners.ts

```diff
@@ -1,30 +1,53 @@
+import {
+  AddContextListenerRequest,
+  AddContextListenerResponse,
+  AddEventListenerRequest,
+  AddEventListenerResponse,
+  AddIntentListenerRequest,
+  AddIntentListenerResponse,
+  AppRequestMessage,
+  PrivateChannelAddEventListenerRequest,
+  PrivateChannelAddEventListenerResponse,
+} from '@kite9/fdc3-schema/generated/api/BrowserTypes';
 import { AutomaticResponse, TestMessaging } from '../TestMessaging';
 import { createResponseMeta } from './support';
 import { v4 as uuidv4 } from 'uuid';
 
+type requests =
```

Comment:
Would you mind pascal casing your types? It'd be consistent with the other types that have been defined. like `type SupportedListenerRequests = ...`